pkg/providers/nutanix/envvars.go

```diff
@@ -8,6 +8,15 @@ import (
 )
 
 func setupEnvVars(datacenterConfig *anywherev1.NutanixDatacenterConfig) error {
+	creds := getCredsFromEnv()
+	if err := os.Setenv(nutanixUsernameKey, creds.username); err != nil {
+		return fmt.Errorf("unable to set %s: %v", nutanixUsernameKey, err)
+	}
+
+	if err := os.Setenv(nutanixPasswordKey, creds.password); err != nil {
+		return fmt.Errorf("unable to set %s: %v", nutanixPasswordKey, err)
+	}
```

Comment:
What purpose does this serve here? `getCredsFromEnv` fetches the credentials from the environment variables that you are trying to set here.

Reply:
initially i thought its required to pass the variables to container where clusterctl init command is actually run. tried without and seems not needed at this time. removed for timebeing, but we need to revisit the SetENvVar function as its called from 3 places and need to understand what all things we need to set here